app/src/main/res/raw/repositories.json

```diff
@@ -89,6 +89,7 @@
   {"name":"PixysOS", "url":"http://gerrit.pixysos.com/"},  
   {"name":"PolarSys", "url":"https://git.polarsys.org/r/", "ci_accounts": "^(Hudson CI)$"},
   {"name":"PotatoProject", "url":"https://review.potatoproject.co/"},
+  {"name":"Project-404", "url":"https://review.project404.co/"},
```

Comment:
"name":"Project-404" must be a valid Java Name. Maybe, "Project404"?

In addition, https://project404.co/ returns a 404 error. It this a public accesible repo?